src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -1385,17 +1451,31 @@ TEST_CASE("custom_json_data: handling of optional tensors")
   ]})"};
 
   size_t num_requests = 4;
+  size_t num_threads = 1;
+
+  const auto& ParameterizeNumThreads{[&]() {
+    SUBCASE("1 thread")
+    {
+      num_threads = 1;
+    }
+    SUBCASE("2 threads")
+    {
+      num_threads = 2;
+    }
+  }};
```

Comment:
So much repeated code in this file 😭

Not on you, but something for us to keep in mind if/when we refactor this.